src/Aquifer.Jobs/appsettings.json

```diff
@@ -22,7 +22,6 @@
     "Notifications": {
         "SendResourceAssignmentNotificationTemplateId": "d-d85f76c6b4d344f5bc8b90b27cc40cc3",
         "SendProjectStartedNotificationTemplateId": "d-7760ec3b5ce34b179384d4783cc1bd81",
-        "SendResourceCommentCreatedNotificationTemplateId": "d-ea84b461ed0f439589098053f8810189"
```

Comment:
After this deploys to prod we can also delete/deactivate this email template in SendGrid.